explorer/interpreter/value.cpp

```diff
@@ -271,6 +299,9 @@ void Value::Print(llvm::raw_ostream& out) const {
     case Value::Kind::NominalClassValue: {
       const auto& s = cast<NominalClassValue>(*this);
       out << cast<NominalClassType>(s.type()).declaration().name() << s.inits();
+      if (s.base().has_value()) {
+        out << ", " << *s.base().value();
```

Comment:
I think a different sigil instead of `,` here would be better. For example, right now we'll print things like:
```
Derived{.derived_field = 5}, Base{.base_field = 7}
```
... which looks OK by itself but will be confusing if we're printing a tuple value. For example:
```
(1, Derived{.derived_field = 5}, Base{.base_field = 7}, 3)
```
... looks like a 4-tuple instead of a 3-tuple. Maybe:
```suggestion
        out << " base " << *s.base().value();
```
... so we get:
```
(1, Derived{.derived_field = 5} base Base{.base_field = 7}, 3)
```

(Much as it might be nice to produce valid Carbon here, `{.derived_field = 5, .base = {.base_field = 7} as Base} as Derived` seems to be less readable as well as being more awkward to print, so I'm not going to ask for that.)